### PR TITLE
Make username required for beat submission

### PR DESCRIPTION
**Pull request by kimroen**
This makes it so you need to provide a username in order to submit a beat.

Part of: [ Iterate beat upload form](https://trello.com/c/3PPZ7s5L/420-iterate-beat-upload-form).

### CODE REVIEW

File: components/BeatForm/schema.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ const schema = object<BeatFormValues>({
     .required('Required'),
   username: string()
     .trim()
-    .min(3, 'Too short'),
+    .min(3, 'Too short')
```
Comment (evenbrenna) [inline components/BeatForm/schema.ts]: I think the min length for a username is 2 (enforced client-side in the app) but there are users like Erlend that have one-letter usernames..

Comment (kimroen) [reply components/BeatForm/schema.ts]: > but there are users like Erlend that have one-letter usernames..

Fun!

Should I remove the requirement, or just set it to 2?

Comment (evenbrenna) [reply components/BeatForm/schema.ts]: Let's set it to 2 and deal with it later if Erlend suddenly starts submitting beats :)

Comment (kimroen) [reply components/BeatForm/schema.ts]: Done!

**Pull request merged**